src/BuildScriptGenerator/Hugo/HugoPlatform.cs

```diff
@@ -72,8 +72,15 @@ public PlatformDetectorResult Detect(RepositoryContext context)
             PlatformDetectorResult detectorResult)
         {
             var manifestFileProperties = new Dictionary<string, string>();
+            manifestFileProperties[ManifestFilePropertyKeys.PlatformName] = detectorResult.Platform;
             manifestFileProperties[ManifestFilePropertyKeys.HugoVersion] = detectorResult.PlatformVersion;
 
+            var oryxAppTypeEnvironmentVar = Environment.GetEnvironmentVariable("Oryx_App_Type");
```

Comment:
instead of each platform doing this should this go in the basebashbuildproperties itself..since this applies to all platforms?

Reply:
yeah I was just about to do this ... thanks